lib/desktop/otp/state.dart

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2023 Yubico.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:logging/logging.dart';
+
+import '../../app/logging.dart';
+import '../../app/models.dart';
+import '../../otp/models.dart';
+import '../../otp/state.dart';
+import '../rpc.dart';
+import '../state.dart';
+
+final _log = Logger('desktop.otp.state');
+
+final _sessionProvider =
+    Provider.autoDispose.family<RpcNodeSession, DevicePath>(
+  (ref, devicePath) => RpcNodeSession(
+      ref.watch(rpcProvider).requireValue, devicePath, ['ccid', 'yubiotp']),
```

Comment:
Only newer keys have `yubiotp` exposed over CCID. All have it over OTP. One needs to be used as a fallback to the other.